tests/unit/integrations/evidently/materializers/test_evidently_profile_materializer.py

```diff
@@ -17,27 +17,21 @@
 from evidently.model_profile import Profile
 from evidently.model_profile.sections import DataDriftProfileSection
 
-from zenml.pipelines import pipeline
-from zenml.steps import step
+from zenml.integrations.evidently.materializers.evidently_profile_materializer import (
+    EvidentlyProfileMaterializer,
+)
 
-
-@step
-def materialize_profile() -> Profile:
-    """Materialize the profile."""
-    return Profile(sections=[DataDriftProfileSection()])
-
-
-@pipeline
-def evidently_pipeline(materializer) -> None:
-    """Test pipeline for the Evidently profile materializer."""
-    materializer()
+from ....test_general import _test_materializer
```

Comment:
Wouldn't this import work as well: `from tests/unit/test_general import _test_materializer`? If so, I would prefer that over `from ....`

Reply:
I wasn't able to get that to work, but maybe I'll try again later.

Reply:
Figured out a way to get this to work 🥳. Still doesn't completely make sense to me why it works that way, but at least it beats the old way I was doing it.